modules/shop/component/otherProductCard.tsx

```diff
@@ -11,6 +11,9 @@ interface ProductCardProps {
 }
 
 const OtherProductCard: React.FC<ProductCardProps> = ({ product }) => {
+
+  const shopName = router.query.shopName || '';
```

Comment:
Why's the shop name allowed to be empty?

Reply:
I eventually fixed that.